src/OpenApi/sample/Controllers/TestController.cs

```diff
@@ -39,6 +40,24 @@ public Ok<CurrentWeather> GetCurrentWeather()
         return TypedResults.Ok(new CurrentWeather(1.0f));
     }
 
+    [Route("/nohttpmethod")]
+    public ActionResult<CurrentWeather> NoHttpMethod()
+        => Ok(new CurrentWeather(-100));
+
+    [HttpQuery] // See https://github.com/dotnet/aspnetcore/issues/61089
+    [Route("/query")]
+    public ActionResult<CurrentWeather> HttpQueryMethod()
+        => Ok(new CurrentWeather(0));
```

Comment:
I've added this to flush out support for HTTP QUERY being added  https://github.com/dotnet/aspnetcore/issues/61089#issuecomment-3143768724.

Alternatively I can just add `HttpMethod.Query` to the switch in this PR as I'm already touching that method code.